File: Java/Events/AvailableEventInfo.java
package LTD_API;

import java.util.List;

public class AvailableEventInfo
{
    private String eventName;
    private int eventId;
    private List<AvailablePerformanceInfo> availablePerformanceInfos;
    
    public String getEventName()
    {
        return eventName;
    }
    public void setEventName(String eventName)
    {
        this.eventName = eventName;
    }

    public int getEventId()
    {
        return eventId;
    }
    public void setEventId(int eventId)
    {
        this.eventId = eventId;
    }

    public List<AvailablePerformanceInfo> getAvailablePerformanceInfos()
    {
        return availablePerformanceInfos;
    }
    public void setAvailablePerformanceInfos(List<AvailablePerformanceInfo> availablePerformanceInfos)
    {
        this.availablePerformanceInfos = availablePerformanceInfos;
    }
}

File: PHP/Events/AvailableEventInfo.php
<?php
namespace LTD_API;

class AvailableEventInfo
{
    private $eventName;
    private $eventId;
    //array of AvailablePerformanceInfo
    private $availablePerformanceInfos;
    
    public function getEventName()
    {
        return $this->eventName;
    }
    public function setEventName($eventName)
    {
        $this->eventName = $eventName;
    }

    public function getEventId()
    {
        return $this->eventId;
    }
    public function setEventId($eventId)
    {
        $this->eventId = $eventId;
    }

    public function getAvailablePerformanceInfos()
    {
        return $this->availablePerformanceInfos;
    }
    public function setAvailablePerformanceInfos($availablePerformanceInfos)
    {
        $this->availablePerformanceInfos = $availablePerformanceInfos;
    }
}
